test/Test/BaseDAO/Management.hs

```diff
@@ -12,16 +12,39 @@ module Test.BaseDAO.Management
 import Universum
 
 import Test.Tasty (TestTree, testGroup)
-
 import Lorentz
-import Lorentz.Contracts.BaseDAO (mkStorage)
--- import Michelson.Untyped.Entrypoints (unsafeBuildEpName)
 import Morley.Nettest
 import Morley.Nettest.Tasty (nettestScenarioCaps)
 import Named (defaults, (!))
-import Test.Common
--- import Tezos.Core (unsafeMkMutez)
-import Util.Named
+
+import qualified Lorentz.Contracts.BaseDAO as DAO
+import Lorentz.Contracts.BaseDAO.Types
+import BaseDAO.ShareTest.Management
+
+-- | Function that originates the contract and also make a bunch of
```

Comment:
```suggestion
-- | Function that originates the contract and also makes a bunch of
```

Reply:
I've made all the changes you suggested now.